tests/utils/integration_test_issue.py

```diff
@@ -0,0 +1,18 @@
+from github import Github
+import os
+import sys
+from datetime import date
+
+token = os.getenv('GIT_TOKEN', '...')
+g = Github(token)
+repo = g.get_repo("radical-cybertools/radical.pilot")
+
+with open(sys.argv[2]) as f:
+    lines = f.readlines()
+
+i = repo.create_issue(
+    title="Integration Tests failed on %s : %s" % (sys.argv[1], date.today()),
+    body='```python\n' + ''.join(x for x in lines) + '\n```',
+    labels=[repo.get_label(name="topic:testing"),
+            repo.get_label(name="type:bug"),
+            repo.get_label(name="layer:rp")])
```

Comment:
Can we get the list of open issues and search for a string in the issue title? If so, we could create a unique ID for each failed-test issue and open a ticket only if that token is not already in any of the issue titles.

Reply:
We can. How do you propose to create that unique ID?

Reply:
I would probably look at https://docs.python.org/3/library/uuid.html. We need information about: test ID, status of the ticket:

Reply:
Creating some ID is certainly doable - but how do we know that a certain failure belongs to a certain ticket?  Adding the error message to the issue maybe, and parse for that error message in all issues labeled `testing` if that error message is logged?

Reply:
I would assume the error message is in the body of the ticket? But the test does not need to know. Each test just check whether its ID is in the subject of any existing ticket. We do need to know the relationship between test, ID and error but that should be all derivable from the ticket itself: ID in the subject; name/location of the test in the body of the message; error in the body of the message. Was this what you were suggesting?

Reply:
> I would assume the error message is in the body of the ticket? But the test does not need to know. 

I am not sure what you mean here? The tests do not know the state of their previous run. `Pytest` runs multiple tests at a time. 

> Each test just checks whether its ID is in the subject of any existing ticket.

A test may fail for different reasons and it is not necessary that the same failed test will fail with the same error.

I think it may make sense to calculate the checksum of error that is included in the body of an issue, and include that in the issue name. If the failure is the same, then we do not need to report a new issue. Would a ping in the issue make sense? And we can ping a team instead of a person.

Reply:
We maybe should not overthink this too much, otherwise we spend time on test fingerprinting and issue automation etc.  The goal is integration testing, and opening issues is a nice bonus.  I would suggest we roll with the 1week/4week cycle we have defined now, and we'll see how it goes.  If in 6 months we have 6 identical issues, we can pick this up and think of something better...

Reply:
OK, let's see how it goes then.